tests/mvp-demo/mvp-demo.go

```diff
@@ -329,9 +331,9 @@ var _ = framework.MvpDemoSuiteDescribe("MVP Demo tests", Label("mvp-demo"), func
 
 	})
 
-	Describe("MVP Demo Chapter 2 - advanced pipeline, JVM rebuild, successful release, switch to simple build", Label("mvp-demo-chapter-2"), Ordered, func() {
+	Describe("MVP Demo Chapter 2 - advanced pipeline, JVM rebuild, successful release", switch to simple build", Label("mvp-demo-chapter-2"), Ordered, func() {
```

Comment:
```suggestion
	Describe("MVP Demo Chapter 2 - advanced pipeline, JVM rebuild, successful release, switch to simple build", Label("mvp-demo-chapter-2"), Ordered, func() {
```
this is causing an error during the [docker build](https://github.com/redhat-appstudio/e2e-tests/actions/runs/5275060688/jobs/9540152690?pr=591)

Reply:
Added that by careless. Removed. Thanks

Reply:
Thanks!